src/test/migrations/steps/utils/runCalculations.ts

```diff
@@ -0,0 +1,69 @@
+import * as pgPromise from 'pg-promise'
+
+import { AssessmentName } from '@meta/assessment'
+
+import { AreaController } from '@server/controller/area'
+import { AssessmentController } from '@server/controller/assessment'
+import { BaseProtocol, Schemas } from '@server/db'
+
+import { calculateRow } from './calculateRow'
+import { getRow } from './getRow'
+
+export const runCalculations = async (
+  props: {
+    assessmentName: AssessmentName
+    cycleName: string
+    metaCache: boolean
```

Comment:
should we pass directly assessment and cycle

Reply:
what about metaCache?

Reply:
I think assessment and cycle can be passed from caller ; in that case metaCache is not needed here

Reply:
> what about metaCache?

should be assessment.metaCache otherwise throw error meta cache missing